views/pages/single_fixture.js

```diff
@@ -413,6 +467,15 @@ function handleChannel(channel, mode) {
       str += '</td>';
 
       str += '</tr>';
+
+      if ('switchToChannels' in cap) {
+        for (let i = 0; i < cap.switchToChannels.length; i++) {
+          const switchingChannel = channel.switchesChannels[i];
+          const switchingChannelIndex = mode.channels.indexOf(switchingChannel);
+          const switchTo = cap.switchToChannels[i];
+          str += `<tr><td colspan="7" class="switch-to-channel"><small>${switchingChannel} (channel ${switchingChannelIndex}) → ${switchTo}</small></td></tr>`;
```

Comment:
why a `<small>` tag when you already have a CSS class where you can change the font-size?